### PR TITLE
[Dashboard] Tool Component Card

### PR DESCRIPTION
**Pull request by christineNg1028**
## Notion ticket link

[Tool Component Card](https://www.notion.so/uwblueprintexecs/Dashboard-Tool-Component-Card-dd51448afabb49cda5c4fe2f358f63ee)

## Implementation description
* Tool component card for the dashboard
* Styling adjustments

Screenshot 2021-07-21 214226
Published Tool

Screenshot 2021-07-21 214205
Unpublished Tool w/ Module

Screenshot 2021-07-21 214140
Unpublished Tool w/o Module (disabled Publish Tool button)

## What should reviewers focus on?
* Ensure meets design criteria
* Ensure functional components
* Provide styling feedback

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: src/components/toolCard/index.tsx [hunk 1]
```diff
@@ -9,11 +9,11 @@ import {
     TagLeftIcon,
     TagLabel,
     Flex,
-    Button,
     IconButton,
     Tooltip,
 } from "@chakra-ui/react";
 import { CheckIcon, DeleteIcon, StarIcon } from "@chakra-ui/icons";
+import { Button } from "..";
```
Comment (mkanoria) [inline src/components/toolCard/index.tsx:16]: let's remove this unused `useState` on line 1 (sorry can't comment on that directly :?)

File: src/components/toolCard/index.tsx [hunk 2]
```diff
@@ -70,49 +70,64 @@ export const ToolCard: React.FC<ToolCardProps> = ({
...
+                                        >
+                                            Link Module
+                                        </Button>
+                                        <Tooltip
+                                            label="Link a module to publish your tool!"
+                                            hasArrow
+                                            placement="top"
+                                        >
+                                            <Button isDisabled _hover="none">
+                                                Publish Tool
```
Comment (mkanoria) [inline src/components/toolCard/index.tsx:108]: We should add an `onPublish` onClick to this button

**Review by mkanoria — COMMENTED** [review]
small comments, styling looks great otherwise!

**Review by mkanoria — APPROVED** [review]
LGTM! Thanks for updating this!

**Review by daniel-sm-yu — APPROVED** [review]

**Pull request merged**